gramps/gen/filters/rules/person/_nobirthdate.py

```diff
@@ -45,7 +45,9 @@ class NoBirthdate(Rule):
     description = _("Matches people without a known birthdate")
     category = _("General filters")
 
-    def apply(self, db, person):
+    def apply_to_one(self, db, data):
+        person = self.get_object(data)
+        # FIXME: can probably do all of this without object:
```

Comment:
use data["birth_ref_index"] to avoid contructing the object?

Reply:
Replaced with:

```
person.event_ref_list[person.birth_ref_index]
```